### PR TITLE
[lbMerge - 1] Remove old, extraneous load balancers

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

**Comment by epmikida** [comment]
Looks good to me, other than the build error, but would this also be a good patch to rename DistributedLB to GrapevineLB?

**Comment by rbuch** [comment]
> Looks good to me, other than the build error, but would this also be a good patch to rename DistributedLB to GrapevineLB?
 
Yeah, could be, but I think I'm more pro having another commit/PR that is responsible for renaming DistributedLB.

File: src/scripts/Makefile [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #     * Charm++ runtime
 #     * ITC++ Translator
 #     * Languages
+#     * Documentation
```
Comment (evan-charmworks) [inline src/scripts/Makefile]: Should this line be here?

Comment (rbuch) [reply src/scripts/Makefile]: No, good catch, I'll take it out.

**Pull request merged**